plugin/src/main/java/dev/lavalink/youtube/plugin/YoutubeRestHandler.java

```diff
@@ -82,6 +82,12 @@ public ResponseEntity<StreamingResponseBody> getYoutubeVideoStream(@PathVariable
                 formats = client.loadFormats(source, httpInterface, videoId);
             } catch (CannotBeLoaded cbl) {
                 throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "This video cannot be loaded. Reason: " + cbl.getCause().getMessage());
+            } catch (FriendlyException t) {
```

Comment:
This should perhaps mirror how the [main class handles non-fatal exceptions.](https://github.com/lavalink-devs/youtube-source/blob/main/common/src/main/java/dev/lavalink/youtube/YoutubeAudioSourceManager.java#L225-L229)